### DISCUSSION TITLE
NequIPCalculator problem

### DISCUSSION THREAD

**Discussion by 7radians** [mir-group/nequip#361] [Q&A] [answered]
Hi, I'm trying do some analysis on the trained Nequip model in Python and need an ASE calculator. `from nequip.ase import NequIPCalculator` gives me "no module nequip.ase exists". While`import nequip` works fine. I found that the ASE integration page in the Nequip docs is empty. Would you be able to point me in the right direction please?

**Comment by DavidW99** [comment, marked as answer]
Hi, Maria
You may find an example to use nequip calculator at (https://github.com/mir-group/nequip/discussions/115). If `import nequip` works, so should `from nequip.ase import NequIPCalculator`. Could you double check if you have loaded the same Python environment for using these two commands?

**Reply by 7radians** [reply]
Thank you for your reply! Yes, I run these two lines from the same python virtual environment. But you somehow solved my problem. I installed nequip on my base environment and `from nequip.ase import NequIPCalculator` now runs just fine. Thanks a lot
